olaf/common/oresat_file_cache.py

```diff
@@ -85,10 +87,9 @@ def remove(self, file_name: str):
         with self._lock:
             for f in self._data:
                 if f.name == file_name:
-                    remove(f.path)
-                    self._data.remove(f)
```

Comment:
This line is still needed, it remove the item that is getting deleted from the list